cloud/scope/cluster.go

```diff
@@ -27,8 +27,10 @@ import (
 	"github.com/Azure/go-autorest/autorest/to"
 	"github.com/go-logr/logr"
 	"github.com/pkg/errors"
+
 	"k8s.io/klog/klogr"
 	"k8s.io/utils/net"
+
```

Comment:
Why the newline? I believe goimports will group together in order with a space between, stdlib, external imports, local imports.